src/sqlfluff/core/linter/linter.py

```diff
@@ -573,6 +573,7 @@ def parse_string(
         recurse: bool = True,
         config: Optional[FluffConfig] = None,
         encoding: str = "utf-8",
+        templated_file: Optional[List[TemplatedFile]] = None
```

Comment:
Optional output parameter: Receives the `TemplatedFile` returned by rendering the raw SQL